README.rst

```diff
@@ -0,0 +1,22 @@
+====================
+djangocms-fil-permissions
+====================
+
+Installation
+============
+
+Requirements
+============
+
+django CMS FIL Permissions requires that you have a django CMS 4.0 (or higher) project already running and set up.
+
+
+To install
+==========
+
+Run::
+
+    pip install djangocms-fil-permissions
+
+Add ``djangocms_fil_permissions`` to your project's ``INSTALLED_APPS``.
```

Comment:
Doesn't it also require adding an auth backend?

Reply:
True, it says so in the actual docs